util.js

```diff
@@ -87,6 +87,24 @@ function tempFile (cached) {
   return cached + '.' + process.pid + '-' + Math.random().toString(16).slice(2) + '.tmp'
 }
 
+function packageOrigin (env, pkg) {
+  // npm <= 6: metadata is stored on disk in node_modules
+  if (pkg._from) {
+    return pkg._from
+  }
+
+  // npm 7: metadata is exposed to environment by arborist
+  if (env.npm_package_from) {
```

Comment:
Should we remove the check for `npm_package_from` and rely on only `npm_package_resolved`?

Reply:
According to the [RFC](https://github.com/npm/rfcs/blob/latest/implemented/0021-reduce-lifecycle-script-environment.md) it should be implemented. I can comment out this code until that's clarified.

Reply:
Happy to keep the `npm_package_from` check, either way it doesn't prevent this PR from being merged.